### ISSUE TITLE
Testing using loggingService?

### ISSUE THREAD

**Issue by dnfield** [dart-lang/appengine#108]
Is there a good way to mock or stub the loggingService when writing unit tests?

**Comment by jonasfj** [comment]
No, but if you're using https://pub.dev/packages/logging then you can avoid setting up AppEngine services when running tests.

Just call [`useLoggingPackageAdaptor`](https://pub.dev/documentation/appengine/latest/appengine/useLoggingPackageAdaptor.html) when using services and logs will get picked up by the logging service.
